RoboJackets/igvc-software
# Feat/actual barrel detection

# Description

Actual Barrel Detention from Lidar data

This PR does the following:
- Create a program that can detect the position of barrels from point cloud data

Fixes #569 

# Testing steps (If relevant)
## Test Case 1
1. Play a rosbag with lidar data
2. Run Josh's ground separation program
3. Run my actual barrel detection program
4. Run rviz to see the result.

Expectation: Green cylinders should show up in rviz.

# Self Checklist
